frontend/src/metabase/core/components/FormInput/FormInput.tsx

```diff
@@ -17,6 +17,7 @@ export interface FormInputProps
   > {
   name: string;
   title?: string;
+  titleActions?: ReactNode;
```

Comment:
maybe just `actions`? I've this property in other components in the app. I don't think we would add actions in another place; and semantically we probably shouldn't be thinking that actions are specifically added near the title

Reply:
I'm fine with any name, your point makes sense to me 👍